x-pack/plugins/ml/public/application/timeseriesexplorer/components/forecasting_modal/forecasting_modal.js

```diff
@@ -18,7 +18,7 @@ import { EuiButton, EuiToolTip } from '@elastic/eui';
 
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n-react';
-import { withKibana } from '@kbn/kibana-react-plugin/public';
+import { withKibana, context } from '@kbn/kibana-react-plugin/public';
```

Comment:
This component now uses both `context` and `withKibana`, IMHO we should try to stick to one. To get rid of `withKibana()` you need to update this line in the file (not part of the diff here):

```
// old withKibana:
      const { timefilter } = this.props.kibana.services.data.query.timefilter;
// update with context:
      const { timefilter } = this.context.services.data.query.timefilter;
```

Then you should be able to remove the last line in the file (`export const ForecastingModal = withKibana(ForecastingModalUI);`) and export this component as `ForecastingModal` directly.

Reply:
Updated in e33fb0d49ce37dcabbd55273fdefa6c425110d67